app/controllers/application_controller.rb

```diff
@@ -2,4 +2,13 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  before_filter :configure_permitted_parameters, if: :devise_controller?
```

Comment:
I also don't like adding this to the application controller, but all the solutions I found at the devise documentation (section strong parameters) implied this.
Let me know if you know a better solutions, e.g that would only affect the school_controller